cinder/mkdocs_theme.yml

```diff
@@ -2,3 +2,12 @@ cinder_theme: true
 
 static_templates:
   - 404.html
+
+include_search_page: false
```

Comment:
What does this new setting do?

Reply:
Copied from: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/themes/mkdocs/mkdocs_theme.yml#L6

You can read what it does here: https://www.mkdocs.org/user-guide/custom-themes/#include_search_page

Reply:
It sounds like this default results in search results being displayed in the modal dialog window.  Sounds good.  That was the previous default behavior.  Ty!